src/components/ProductThumbnail.vue

```diff
@@ -147,6 +147,14 @@ export default {
     originalPrice() {
       return this.matchingVariant.price.value;
     },
+
+    productSlug() {
+      return this.currentProduct.slug;
+    },
+
+    sku() {
+      return this.matchingVariant.sku;
```

Comment:
I'm not sure it's always the same structure, but any method that you see repeated across components we should move it into its own mixin to make sure that we never have repeated code. For example, `currentProduct` I think it appears several times, right?